inc/search.class.php

```diff
@@ -551,6 +623,12 @@ static function prepareDatasForSearch($itemtype, array $params, array $forcedisp
          }
       }
 
+      /*if (!count($data['toview'])) {
+         Toolbox::logWarning(
+            'No columns found to display ' . $itemtype
+         );
+      }*/
+
```

Comment:
```suggestion
```

Reply:
No, I guess we should keep that one; I commented it because I had failures on unit tests, but a search returning no columns should really not happen.